setup.cfg

```diff
@@ -11,7 +11,7 @@ excludes=.tox,utils,files
 lint_disable=fixme,locally-disabled,file-ignored,duplicate-code
 
 [flake8]
-exclude=.tox/*,utils/*,inventory/*
+exclude=.tox/*,utils/*,inventory/*,roles/lib_utils/src/*,roles/lib_openshift/src/*
```

Comment:
> This may suffer from the same specificity x maintainability we discussed w.r.t. test discovery.

> What is inventory/*,roles/lib_utils/src/*,roles/lib_openshift/src/* giving us (in terms of speed, false positives, etc)?

I added roles/lib_{openshift,utils}/src/* to avoid the need to add `# flake8: noqa` and `# noqa: E302,E301 at the top of every file. 

If we can make flake8 (and ideally pylint as well) work against the src files, I would definitely prefer that than the current situation.

> I do understand that getting a flake8 error in both src and then 20-something generated files is unnecessarily confusing. The trouble is if we get it only on the 20 generated files, it is less convenient to find the source of the error and fix it in the source, then regenerate.

I'm not sure we get away from that, since we are just blanket ignoring the files currently.

> Imagine, the report will say a file name and line, but you actually need to go edit a different file in a completely different line... #contributor-experience :-)

Agreed, which is why I would like to see if we can rework the code generation in a way that the src files could be tested independently.